wg-charter.html

```diff
@@ -0,0 +1,406 @@
+
+        <dt><a href="https://www.w3.org/2011/webappsec/">Web Application Security Working Group</a></dt>
+        <dd>This Working Group develops security and policy mechanisms to improve the security of Web Applications, and enable secure cross-site communication.</dd>
+
+        <dt><a href="https://www.w3.org/wasm/">Web Assembly Working Group</a></dt>
+        <dd>This Working Group develops a size- and load-time-efficient format and execution environment, allowing compilation to the web with consistent behavior across a variety of implementations.</dd>
+
+        <dt><a href="https://www.w3.org/community/webmachinelearning/">Machine Learning for the Web Community Group</a></dt>
+        <dd>This Community Group incubates a dedicated low-level Web API for machine learning inference.</dd>
+
+        <dt><a href="https://www.w3.org/community/wicg/">Web Platform Incubator Community Group</a></dt>
+        <dd>
+          This group provides a lightweight venue for proposing, incubating and discussing new web platform features. The Web Performance Working group will incubate and review new proposals that are within scope of our charter within the WICG. Once such WICG-incubated proposal is implemented and available in at least one major browser, and has support from one more, it may be adopted by the Web Performance Working group.
+        </dd>
+          </dl>
+          <h3 id="external-coordination">External Organizations</h3>
+          <dl>
+            <dt><a href="https://www.ecma-international.org/memento/tc39.htm" id="tc39">ECMA Technical Committee 39 (TC39)</a></dt>
+          <dd>This is the group responsible for ECMAScript standardization. As the Web Performance Working Group will be developing ECMAScript APIs, it should collaborate with TC39.</dd>
+          <dt><a title="WHATWG" href="https://whatwg.org/" id="whatwg">Web Hypertext Application Technology Working Group (WHATWG)</a></dt>
+          <dd>The Web Hypertext Application Technology Working Group (WHATWG) is a community of people interested in evolving the web through standards and tests.</dd>
+          <dt><a title="Khronos" href="https://khronos.org/" id="khronos">Khronos</a></dt>
+          <dd>The Khronos Organization is a standards body that develops many graphics-related technologies, including OpenGL and Vulkan.</dd>
+          </dl>
+        </div>
+      </section>
+
+
+
+      <section class="participation">
+        <h2 id="participation">
+          Participation
+        </h2>
+        <p>
+          To be successful, this Working Group is expected to have 5 or more active participants for its duration, including representatives from the key implementors of this specification, and active Editors and Test Leads for each specification. The Chairs, specification Editors, and Test Leads are expected to contribute half of a day per week towards the Working Group. There is no minimum requirement for other Participants.
```

Comment:
What is a "Test Lead"? (This isn't a role I'm familiar with.)

Reply:
Some groups choose to nominate test leads to lead the development of the test suites. That is not as formal a role as editor, and not a mandatory role either. The (boilerplate) text is more meant to convey the fact that work on test suites is part of success criteria for a group.

Reply:
Fair enough.